src/api/serverApi.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+
+const checkServerStatus = () => axios.get(process.env.REACT_APP_STATUS_URL);
+
+export default checkServerStatus;
```

Comment:
The file name does not correspond to the returned type. You could wrap this function in an object and then decompose when importing.